app/views/users/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<h2> List of Users </h2>
+<table>
+  <tr>
+    <th>Username</th>
+    <th>Votes</th>
+    <th>Joined</th>
+  </tr>
+  <% @users.each do |user| %>
+  <tr>
+    <td><%= link_to user.name, user_path(user.id) %></td>
+    <td><%= user.name.capitalize %></td>
+    <%= puts user.join_date %>
```

Comment:
I'm a little curious about the purpose of this `puts`. `puts` will _always_ print to the console, which is useful for debugging, but doesn't help you generate HTML.